AWSAPIGatewayTests/AWSLambdaMicroserviceClient.m

```diff
@@ -130,7 +130,7 @@ - (instancetype)initWithConfiguration:(AWSServiceConfiguration *)configuration {
     if (self = [super init]) {
         _configuration = [configuration copy];
 
-        NSString *URLString = @"https://mykov6r7rh.execute-api.us-east-1.amazonaws.com/prod";
+        NSString *URLString = @"https://yjzm4utpcl.execute-api.us-east-1.amazonaws.com/prod";
```

Comment:
We need this to be parameterized rather than hardcoded. I believe this file is generally code-generated and downloaded, but we can extend it slightly since we're using it as a test fixture.

Reply:
modified the default client to require endpoint url for initialization